tests/unit/Compute/v2/Models/ServerTest.php

```diff
@@ -397,4 +398,13 @@ public function test_it_detaches_volume()
 
         $this->server->detachVolume($attachmentId);
     }
+
+    public function test_it_lists_interface_attachments()
+    {
+        $this->setupMock('GET', 'servers/serverId/os-interface', null, [], 'server-interface-attachments-get');
+
+        $interfaceAttachments = iterator_to_array($this->server->listInterfaceAttachments());
+
+        $this->assertInstanceOf(InterfaceAttachment::class, $interfaceAttachments[0]);
```

Comment:
can we check some field vals too, just to make sure the object is being hydrated properly